typescript/core/src/common/detectVersion.ts

```diff
@@ -1,7 +1,6 @@
-import { MCAP_MAGIC } from "../pre0/constants";
 import { MCAP0_MAGIC } from "../v0/constants";
 
-export type McapVersion = "pre0" | "0";
+export type McapVersion = "0";
```

Comment:
Should we remove this type and replace detectVersion with something like detectMcap or validateMagic? The "detect version" wording doesn't make much sense if we only acknowledge the existence of one version.

Reply:
Yea - I think we should simplify this down to one version. The v0 idea was compelling but its the version we are gonna have for a while. One day we can introduce the subpaths if we get a v1.